### PR TITLE
Update dependencies and change metrics port

### PR DESCRIPTION
**Pull request by sudermanjr**
For some reason the metrics port was always in use when it tried to use 8080. I have moved it to 8042, and that seems to work.

### CODE REVIEW

**Review by lucasreed — APPROVED** [review]
Code all looks good, I'm curious though if we can remove the CRD definition from `deploy/all.yaml` since you added in the `deploy/crd.yaml` file?

**Comment by sudermanjr** [comment]
> Code all looks good, I'm curious though if we can remove the CRD definition from `deploy/all.yaml` since you added in the `deploy/crd.yaml` file?

Good question. My plan was to leave both, so that people could pick `all` or the `crd`, but I'm open to either way.

**Comment by lucasreed** [comment]
> > Code all looks good, I'm curious though if we can remove the CRD definition from `deploy/all.yaml` since you added in the `deploy/crd.yaml` file?
> 
> Good question. My plan was to leave both, so that people could pick `all` or the `crd`, but I'm open to either way.

I'll leave it up to you before merging this. Only concern really is needing to update two files when the CRD changes. We can also make a decision when that happens next.

**Comment by sudermanjr** [comment]
@lucasreed I liked splitting it all up, but didn't like the name all.yaml, so I split out all the stuff into separate yaml files and then updated the e2e tests.

File: e2e/pre.sh
Comment (lucasreed) [inline e2e/pre.sh:15]:
```suggestion
yq w -i deploy/3_deployment.yaml 'spec.template.spec.containers[0].image' "quay.io/reactiveops/rbac-manager:dev-$CI_SHA1"
```

I think this should fix it.

**Review by lucasreed — CHANGES_REQUESTED** [review]

File: e2e/pre.sh
Comment (sudermanjr) [reply e2e/pre.sh:15]: Ah! Thanks!

**Review by lucasreed — APPROVED** [review]

**Comment by lucasreed** [comment]
@sudermanjr One thing I just thought of. I think a corresponding PR to the charts repo is necessary here as well. The various port numbers in this block of code will need updating to `8042`: https://github.com/FairwindsOps/charts/blob/master/stable/rbac-manager/templates/deployment.yaml#L43-L69

**Comment by sudermanjr** [comment]
> @sudermanjr One thing I just thought of. I think a corresponding PR to the charts repo is necessary here as well. The various port numbers in this block of code will need updating to `8042`: https://github.com/FairwindsOps/charts/blob/master/stable/rbac-manager/templates/deployment.yaml#L43-L69

yeah, I was planning to do that as soon as we planned a release of this.

**Pull request merged**
